test/res.sendFile.js

```diff
@@ -615,7 +615,7 @@ describe('res', function(){
         .get('/')
         .end(function(err, res){
           res.text.should.equal('<p>{{user.name}}</p>');
-          res.headers.should.have.property('content-type', 'text/html; charset=UTF-8');
+          should(res.headers).have.property('content-type', 'text/html; charset=UTF-8');
```

Comment:
If that is the case, would you be OK if I simply fixed these to actually use supertest instead for the assertions, which corrects the underlying issue over accepting this pull request? I just asked because I don't want to flat out reject a pull request since it achieves a purpose, but the actual changes made here just don't align with the current practices of this module or these other modules in Express. I am happy to work through you to make the necessary changes if you want your PR to be accepted, though. Let me know how you would like to proceed.